packages/build/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/build",
-  "version": "0.5.0",
+  "version": "0.5.0-fb-dependabotPackageUpdates.0",
```

Comment:
Should we go ahead and mark this as version 1.0?

Reply:
Probably not the best reason, but I was holding off on going to 1.0 until I get that last internal module/package converted: assayreport. Hoping to do that today or tomorrow.